cashctrl_ledger/tests/test_price_history.py

```diff
@@ -0,0 +1,16 @@
+"""Test suite for price history operations."""
+
+import pytest
+from pyledger.tests import BaseTestPriceHistory
+# flake8: noqa: F401
+from base_test import initial_engine
+
+
+class TestPriceHistory(BaseTestPriceHistory):
+
+    @pytest.fixture
+    def engine(self, initial_engine):
+        # Need to clear price history before tests.
+        # Initial engine keeps one price history file across whole module tests
```

Comment:
I do not understand this comment.
- What is the behaviour, and why did we choose this behaviour?
- Is this still a clever choice now that we had re-factored the price lookup?

Reply:
If we go through the steps, that would be more cleaner:

- In the `initial_engine,` we instantiate the `ExtendedCashCtrlLedger` class, which creates the `Price.csv` file. This fixture's scope is `scope="module"` which means this file will persist until the end of all tests.
- In the `engine` fixture we need to clear prices to make the `Price.csv` file pristine before each test run.

I believe we can even drop the comment there since it is intuitive to have pristine prices before each test.

Reply:
Agreed.